source/future.tex

```diff
@@ -115,6 +115,19 @@
 \end{codeblock}
 \end{example}
 
+\rSec1[depr.ellipsis.comma]{Non-comma-separated ellipsis parameters}
+
+A \grammarterm{parameter-declaration-clause}
+of the form
+\grammarterm{parameter-declaration-list} \tcode{...}
```

Comment:
Likewise, I think this should have an iref back to [dcl.fct] (I think that is the right clause)